test/integration/test-example-validation.js

```diff
@@ -54,6 +54,7 @@ describe('example', function() {
     'instagram.amp.html',
     'released.amp.html',
     'soundcloud.amp.html',
+    'springboard-player.amp.html',
```

Comment:
@cramforce are the tests failing because this was added and the validator doesn't have springboard yet?